api/src/main/java/io/kafbat/ui/service/ApplicationInfoService.java

```diff
@@ -44,36 +45,49 @@ public ApplicationInfoService(DynamicConfigOperations dynamicConfigOperations,
                                 ApplicationContext applicationContext,
                                 @Autowired(required = false) BuildProperties buildProperties,
                                 @Autowired(required = false) GitProperties gitProperties,
+                                @Value("${" + GITHUB_RELEASE_INFO_ENABLED + ":true}") boolean githubInfoEnabled,
                                 @Value("${" + GITHUB_RELEASE_INFO_TIMEOUT + ":10}") int githubApiMaxWaitTime) {
     this.applicationContext = applicationContext;
     this.dynamicConfigOperations = dynamicConfigOperations;
     this.buildProperties = Optional.ofNullable(buildProperties).orElse(new BuildProperties(new Properties()));
     this.gitProperties = Optional.ofNullable(gitProperties).orElse(new GitProperties(new Properties()));
-    githubReleaseInfo = new GithubReleaseInfo(githubApiMaxWaitTime);
+    if (githubInfoEnabled) {
+      this.githubReleaseInfo = new GithubReleaseInfo(githubApiMaxWaitTime);
+    } else {
+      this.githubReleaseInfo = null;
+    }
   }
 
   public ApplicationInfoDTO getApplicationInfo() {
-    var releaseInfo = githubReleaseInfo.get();
+    var releaseInfo = githubReleaseInfo != null ? githubReleaseInfo.get() : null;
     return new ApplicationInfoDTO()
         .build(getBuildInfo(releaseInfo))
         .enabledFeatures(getEnabledFeatures())
         .latestRelease(convert(releaseInfo));
   }
 
   private ApplicationInfoLatestReleaseDTO convert(GithubReleaseInfo.GithubReleaseDto releaseInfo) {
+    if (releaseInfo == null) {
+      return null;
```

Comment:
`@Nullable` for the method please, or a non-null approach

Reply:
Any should work fine, there's no guideline on this rn, feel free to use any of these